amethyst_core/src/lib.rs

```diff
@@ -44,6 +44,7 @@ pub use self::{
 };
 
 pub mod bundle;
+pub mod deferred_dispatcher_operation;
```

Comment:
would prefer the `deferred_` prefix to not be there (am pedantic like that :upside_down_face:)

Reply:
I've asked @jojolepro to understand the correct naming, so I suppose that `dispatcher_operation.rs` would make for sense for you.

But I don't like it, it give me the impression that is meaningless. And by the way I don't like the actual one either.

Do you have any other idea?

Reply:
i named the enum so :stuck_out_tongue:
i don't mind that much / can let it go :snowflake: